### ISSUE TITLE
TypeError in FreqDist.tabulate for bigrams

### ISSUE THREAD

**Issue by danuker** [nltk/nltk#2334]
I use the following code:

```python
with open(corpus_filename) as corpus_f:
    tokens = preprocess(corpus_f.read())
    bgs = nltk.bigrams(tokens)
    fdist =  nltk.FreqDist(bgs)
    print(fdist.tabulate())
```

What happens is the `tabulate` method can't format the tuples generated by `bigrams`:
`width = max(len("%s" % s) for s in samples)`

When it gets to any key like `('hello', 'world)`, it breaks:
```python
Traceback (most recent call last):
  File "learner.py", line 32, in <module>
    print(fdist.tabulate())
  File "<edited>/env/lib/python3.7/site-packages/nltk/probability.py", line 333, in tabulate
    width = max(len("%s" % s) for s in samples)
  File "<edited>/env/lib/python3.7/site-packages/nltk/probability.py", line 333, in <genexpr>
    width = max(len("%s" % s) for s in samples)
TypeError: not all arguments converted during string formatting
```

This is because the string formatting doesn't work:
```bash
$ ipython
Python 3.7.3 (default, Apr  3 2019, 05:39:12) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.6.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: "%s" % ('hello', 'world')                                                                                                                                                             
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-60c0688ae365> in <module>
----> 1 "%s" % ('hello', 'world')

TypeError: not all arguments converted during string formatting

In [2]: "{}".format(('hello', 'world'))                                                                                                                                                       
Out[2]: "('hello', 'world')"
```

Would it be OK to replace the `%s` formatting with the ["new style" string formatting operator](https://pyformat.info/)?

**Comment by EeshitaBiswas** [comment]
Can I start working on this?

**Comment by danuker** [comment]
@EeshitaBiswas if you ask me, I would say yes, it seems @stevenbird is assigned to this, but did not respond, so he is busy, and I don't think this task is too hard.

**Comment by tomaarsen** [comment]
This has been resolved in 52228dfcbff8ec3446fd87a3ee0d8cea05db5adf. Thank you for raising this issue!
